### ISSUE TITLE
Package versions in Manifest may not be compatible with Julia 1.1

### ISSUE THREAD

**Issue by denizyuret** [KnetML/MAC-Network#3]
I get the following error in cell 5:
```
Error encountered while loading "/mnt/efs/comp541/MAC-Network/data/demo/CLEVR_v1.0/images/val/CLEVR_val_013382.png".
Fatal error:
rethrow(exc) not allowed outside a catch block

Stacktrace:
 [1] rethrow(::InitError) at ./error.jl:52
 [2] handle_error(::InitError, ::FileIO.File{FileIO.DataFormat{:PNG}}) at /home/ec2-user/.julia/packages/FileIO/lt9I0/src/error_handling.jl:80
 [3] handle_exceptions(::Array{Any,1}, ::String) at /home/ec2-user/.julia/packages/FileIO/lt9I0/src/error_handling.jl:75
 [4] #load#27(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::FileIO.File{FileIO.DataFormat{:PNG}}) at /home/ec2-user/.julia/packages/FileIO/lt9I0/src/loadsave.jl:193
 [5] load(::FileIO.File{FileIO.DataFormat{:PNG}}) at /home/ec2-user/.julia/packages/FileIO/lt9I0/src/loadsave.jl:172
 [6] #load#13(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::String) at /home/ec2-user/.julia/packages/FileIO/lt9I0/src/loadsave.jl:118
 [7] load(::String) at /home/ec2-user/.julia/packages/FileIO/lt9I0/src/loadsave.jl:118
 [8] top-level scope at In[5]:1
```

**Comment by denizyuret** [comment]
Apparently ImageMagick needs to be loaded first before anything else. PR #4 fixes this.

**Comment by ekinakyurek** [comment]
Thank you. I think you also have different updates for visualization in
this :
https://github.com/KnetML/NLPdemos/commit/6b8d8a416ca18c1e1642d6b91efef4a8c5874782

Could you add them too?

**Comment by ekinakyurek** [comment]
Oh, I think you’ve done already?

**Comment by denizyuret** [comment]
Yes, I changed the visualize() to match NLPdemos.

**Comment by ekinakyurek** [comment]
solved with PR#4
